doc/docs/master/hub/manage-entities/manage-workspaces.md

```diff
@@ -0,0 +1,73 @@
+# Manage Workspaces
+
+In Pachyderm Hub, you run your data science pipelines in workspaces.
+A workspace is a collection of infrastructure resources that run
+Pachyderm components needed to process your data. Workspace resource
+limitations are defined by your account type and provisioned automatically.
+For example, in the free tier, you can only create one workspace, while
+a Pro account allows you to create multiple workspaces.
+
+Each workspace has isolated computing, processing, storage, and other
+resources. Resource autoscaling is managed by Pachyderm behind the
+scenes. You cannot move data or pipelines between workspaces.
+Each workspace in an organization must have a unique name.
+
+## Create a Workspace
+
+You need a workspace to star building your pipelines and working
+with Pachyderm. 
+
+To create a workspace, complete the following steps:
+
+1. Click **Create New Workspace** or **Create a 4-Hour Workspace**.
+1. Fill out the required fields and accept Terms and Conditions.
+1. Click **Create**.
+
+   Your workspace should be created instantly.
+
+## Connect to Your Workspace
+
+You can access your workspace either by using the command-line interface (CLI)
+called `pachctl` or by using the PachDash UI. Although some of the
```

Comment:
Should not mention UI is most of this flow. They MUST connect via pachctl to get any real use out of the system. Dash is a small bonus that we can mention later. 

Use the good phrasing I mentioned in getting started here. No need to write basically the same info twice differently.